pyphare/pyphare/pharesee/run.py

```diff
@@ -108,15 +110,19 @@ def _divB2D(Bx, By, xBx, yBy):
     return dxbx + dyby
 
 
-def _compute_divB(patch):
-    if patch.box.ndim == 1:
+def _compute_divB(patchdatas, **kwargs):
+
```

Comment:
break here implies this file has not been formatted by black?

Reply:
not sure what you mean I have black on save and this is version 22.8

Reply:
doesn't look like it 
![Uploading image.png…]()